Bingha/Bingha/Views/Iceberg/IcebergViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  IcebergViewController.swift
+//  Bingha
+//
+//  Created by 이재웅 on 2022/07/15.
+//
+
+import UIKit
+
+class IcebergViewController: UIViewController {
+    @IBOutlet var roundedRectangle: UIView!
+    @IBOutlet var levelLabel: UILabel!
+    @IBOutlet var reducedCarbonLabel: UILabel!
+    @IBOutlet var informationLabel: UILabel!
+    
+    var circularProgressBarView: CircularProgressBarView!
+    var circularViewDuration: TimeInterval = 2
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setRoundedRectangle()
+        setUpCircularProgressBarView()
+        setLevelLabel()
+        setReducedCarbonLabel()
+        setInformationLabel()
+    }
+    
+    func setRoundedRectangle() {
+        roundedRectangle.layer.cornerRadius = 20
+    }
+    
+    // TODO: - 경험치에 따라 levelLabel 동적으로 변하게
+    func setLevelLabel() {
+        levelLabel.text = "Lv.1"
+        levelLabel.font = .rounded(ofSize: 20, weight: .bold)
+    }
+    
+    func setReducedCarbonLabel() {
+        reducedCarbonLabel.font = .rounded(ofSize: 48, weight: .bold)
+    }
+    
+    func setInformationLabel() {
+        informationLabel.font = .rounded(ofSize: 16, weight: .bold)
+    }
+    
+    func setUpCircularProgressBarView() {
+        circularProgressBarView = CircularProgressBarView(frame: .zero)
```

Comment:
```suggestion
       lazy var circularProgressBarView = CircularProgressBarView(frame: .zero)
```
IcebergViewController 내에서 circularProgressBarView를 선언해줄때 옵셔널타입으로 선언하지 않고 다음과 같이 선언 후 setUpCircularProgressBarView() 함수 내에서는 circularProgressBarView의 layout관련 코드만 작성해주는 것도 하나의 방법이 될것같아 제안드립니다!
(코드 한 줄 덜 작성할 수 있지 않을까 싶은 마음)

Reply:
두 줄이 한 줄로 줄어들어 더 깔끔해 보이네요~!
추가적으로 여기에 lazy 키워드를 붙여주는 이유가 있을까요?